src/lib/components/event-table.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+  import { setContext } from 'svelte';
+
+  import Event from './event.svelte';
+  import VirtualList from '@sveltejs/svelte-virtual-list';
+
+  export let events: (HistoryEventWithId | PendingActivity)[];
+</script>
+
+<section
+  class="flex flex-col border-2 border-gray-300 rounded-lg w-full event-history"
+>
+  <div class="flex w-full">
+    <header class="table-header border-r-2 rounded-tl-lg w-1/3">
+      <h3>Summary</h3>
+      <slot name="filters" />
+    </header>
+    <header class="table-header rounded-tr-lg w-2/3">
+      <h3>Details</h3>
+    </header>
+  </div>
+  <div class="flex h-full overflow-y-hidden">
+    <div
+      class="flex flex-col h-full w-1/3 border-r-2 border-gray-300 rounded-bl-lg"
+    >
+      <div class="h-full rounded-bl-lg overflow-y-scroll">
+        <VirtualList items={events} let:item>
+          <Event event={item} />
+        </VirtualList>
+      </div>
+    </div>
+    <div class="flex flex-col h-full w-2/3">
+      <div
+        class="h-full  overflow-y-scroll overflow-x-hidden rounded-br-lg px-4"
+      >
+        <slot name="details">
+          <div class="flex items-center justify-center w-full h-full">
+            <p class="text-gray-600 italic">(Nothing selected.)</p>
+          </div>
+        </slot>
+      </div>
+    </div>
+  </div>
+</section>
+
+<style lang="postcss">
+  .event-history {
+    height: calc(100vh - 360px);
```

Comment:
I don't love this, but 100% in CSS is "100% _or_ fit to the size of the content. This gets the right effect, but it will need to be adjusted if the content above it changes in size.

Reply:
This is the the fix for workflows as well i agree we should find another way later on